origin-bridge/logic/attestation_service.py

```diff
@@ -460,10 +464,25 @@ def verify_airbnb(eth_address, airbnbUserId):
                 " has not been found in user's Airbnb profile."
             )
 
-        # TODO: determine the schema for claim data
-        data = 'airbnbUserId:' + airbnbUserId
+        ipfs_hash = ipfs_helper.pin_json({
+            'schemaId': 'http://schema.originprotocol.com/airbnb-attestation_v1.0.0',
+            'airbnb_user_id': airbnbUserId
+        })
+
+        """ - IPFS hash is a base58 encoded string
+            - We store IPFS hashes in solidity claims in byte32 hex encoding to minimise
+              gas cost.
+            - byte32 is not string serialisable so it can not be transmitted in that form
+              from discovery to the DApp
+            - discovery needs to transform ipfs hash to base32 hex encoding (that is how
```

Comment:
nits:
 - it is not discovery, it is the bridge server.
 - there is no such thing as base32 encoding. it is just encoded in hexadecimal

Reply:
👍 
Changing the line to: 
```
- bridge needs to transform ipfs hash to bytes32 format (that is how
     it is going to be stored in the contract) before signing the claim, and then
     send IPFS hash to the DApp in base58 string encoding.
```